src/views/Portal/components/ApplicationForm/PriorExperience/index.view.tsx

```diff
@@ -10,50 +10,48 @@ const ExperiencePage: React.FC = () => {
     useApplication()!
   return (
     <div className='experience-page'>
-      <div className='experience-page__container'>
-        <div className='experience-page__container--title'>
-          Prior Experience
-        </div>
-        <TextField
-          name='hackathonCount'
-          handleChange={(e: any) =>
-            setPriorExperienceFormData(prev => ({
-              ...prev,
-              hackathonCount: e.target.value,
-            }))
-          }
-          fieldState={priorExperienceFormData.hackathonCount}
-          errorMessage={priorExperienceFormData.hackathonCountErr}
-          label='hackathonCount'
-          maxLength={3}
-        />
-        <RadioForm
-          question='Will this be your first time attending CruzHacks?'
-          name='firstCruzHacks'
-          inputs={[{ label: "Yes" }, { label: "No" }]}
-          handleChange={(e: any) =>
-            setPriorExperienceFormData(prev => ({
-              ...prev,
-              firstCruzHacks: e.target.value,
-            }))
-          }
-          checkedState={() => {}}
-          errorMessage={priorExperienceFormData.hackathonCountErr}
-        />
-        <TextBox
-          name='Do you have prior tech experience? If so, in what context (i.e., classes, internships, personal projects)?'
-          handleChange={(e: any) =>
-            setPriorExperienceFormData(prev => ({
-              ...prev,
-              priorExperience: e.target.value,
-            }))
-          }
-          fieldState={priorExperienceFormData.priorExperience}
-          errorMessage={priorExperienceFormData.priorExperienceErr}
-          label='priorExperience'
-          maxLength={250}
-        />
-      </div>
+      Prior Experience
+      <RadioForm
+        question='Will this be your first time attending CruzHacks?'
+        name='firstCruzHacks'
+        inputs={[{ label: "Yes" }, { label: "No" }]}
```

Comment:
The validation will always fail for this since it's expecting lowercase "yes" or "no". See below for reference.